src/auth/ui/vue/show.ts

```diff
@@ -136,6 +139,59 @@ export class AuthWebview extends VueWebview {
         }
     }
 
+    async getIdentityCenterRegion(): Promise<Region> {
+        return showRegionPrompter()
+    }
+
+    async startIdentityCenterSetup(startUrl: string, regionId: Region['id']) {
+        try {
+            await CodeWhispererAuth.instance.connectToEnterpriseSso(startUrl, regionId)
+        } catch (e) {
+            // This scenario will most likely be due to failing to connect from user error.
+            // When the sso login process fails (eg: wrong url) they will come back
+            // to the IDE and cancel the 'waiting for browser response'
+            if (CancellationError.isUserCancelled(e)) {
+                return
+            }
+        }
```

Comment:
Seems like this error should be bubbled up and shown in the webview if they didn't cancel?